### DISCUSSION TITLE
Your toolkit works perfectly fine with my device, but it shows the prompt of "The Device is not tested" something.

### DISCUSSION THREAD

**Discussion by singhantariksh** [BartoszCichecki/LenovoLegionToolkit#1130] [Q&A] [answered]
![Screenshot 2024-01-20 221420](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/94468223/b52bb68c-4836-474e-b47c-6f57db987a08)
This is my system. lenovo loq arc a350m with i5 12450x... I've thoroughly tested the available features and everything works just fine, although disabling gpu with the shortcut isn't available as it's a nvidia only feature (I read) but mux switch control does it just as intended, even Vantage wasn't able to do mux tasks perfectly, but this toolkit works with the modes as intended. So if needed, you can update the device compatibility and ask me any further question.
Cheers
Sam

**Comment by BartoszCichecki** [comment, marked as answer]
Moved to #1131
